build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -651,16 +645,16 @@ CONFIG_NET_ARP_TABLESIZE=10
 CONFIG_NET_ARP_QUEUEING=y
 CONFIG_NET_ETHARP_TRUST_IP_MAC=y
 CONFIG_NET_ETH_PAD_SIZE=0
-# CONFIG_NET_ARP_STATIC_ENTRIES is not set
+CONFIG_NET_ARP_STATIC_ENTRIES=y
 CONFIG_NET_UDP=y
 # CONFIG_NET_NETBUF_RECVINFO is not set
 CONFIG_NET_UDP_TTL=255
-CONFIG_NET_UDPLITE=y
+# CONFIG_NET_UDPLITE is not set
 CONFIG_NET_TCP=y
 CONFIG_NET_TCP_TTL=255
 CONFIG_NET_TCP_WND=26280
 # CONFIG_NET_WND_SCALE is not set
-CONFIG_NET_TCP_MAXRTX=12
+CONFIG_NET_TCP_MAXRTX=6
```

Comment:
Please change 
CONFIG_NET_TCP_MAXRTX=12
CONFIG_NET_TCP_SYNMAXRTX=6

to 

CONFIG_NET_TCP_MAXRTX=7
CONFIG_NET_TCP_SYNMAXRTX=7

Reply:
Please let me know if you need these changes in loadable_apps only or all defconfigs?

Reply:
we need to change all defconfig.